### DISCUSSION TITLE
Compatibility with Coulbourn Instruments peripherals

### DISCUSSION THREAD

**Discussion by psynaptix1** [pyControl/code#111] [Q&A]
Hi all,
I was wondering if there was a way to possibly use the pyControl interface to control peripherals such as the ones from Coulbourn Instruments? These peripherals are built to interface via a RJ12 cable with 28 volts.

**Comment by ThomasAkam** [comment]
Hi @psynaptix1,

Apologies for the delay in getting to this.  Currently there is no dedicated pyControl hardware for interfacing with 28V peripherals.   Sussex Neuroscience have done some work on developing interfaces between microcontroller hardware and 28V systems, see: https://github.com/Sussex-Neuroscience/med-associates-bits.  That repository does not have a lot of documentation so you might want to contact the developer @amchagas for more information. 

Thomas

**Reply by amchagas** [reply]
Hi @psynaptix1 and @ThomasAkam,

We developed bits and pieces to interface with med associates systems which also run on 28v. I am happy to chat further about possible collabs for interfacing with another 28v system.. or at least to give pointers and tips on how we implemented stuff here... 

And sorry about the lack of documentation on that repository...

**Reply by ThomasAkam** [reply]
Thanks @amchagas,  I would like to implement an interface for 28v systems at some point, so will take a look at your repository and get in touch if I have any questions.

T
